ginga/util/iohelper.py

```diff
@@ -18,8 +18,10 @@ def get_fileinfo(filespec, cache_dir='/tmp', download=False):
     """
     Parse a file specification and return information about it.
     """
-    idx = None
-    name_ext = ''
+    # Loads first science extension by default.
+    # This prevents [None] to be loaded instead.
+    idx = ('SCI', 1)
+    name_ext = '[SCI,1]'
```

Comment:
@pllim, I had a chance to test this today.  It fails to load any FITS file that does not have:
1) an extension
2) an extension named "SCI"

This includes many FITS files (most of our instruments, actually).  I think what we want to do is make the "canonical name" be the first HDU that can be opened.  So for HST files, if the first extension that can be opened is "[SCI, 1]" that becomes the suffix.  I'll work on this a bit more.